presto-iceberg/src/test/java/com/facebook/presto/iceberg/IcebergDistributedSmokeTestBase.java

```diff
@@ -57,6 +58,11 @@ protected QueryRunner createQueryRunner()
         return IcebergQueryRunner.createIcebergQueryRunner(ImmutableMap.of(), catalogType);
     }
 
+    protected String getLocation(QueryRunner queryRunner, String schema, String table)
```

Comment:
Why was queryRunner a parameter instead of acquiring it in the function implementations? We could either make this method a static one, or call the getQueryRunner() in the implementation. Do you think it's doable?

Reply:
Also, would you please move this protected function to be after the public ones and overriden protected ones in all 4 classes?

Reply:
> Why was queryRunner a parameter instead of acquiring it in the function implementations? We could either make this method a static one, or call the getQueryRunner() in the implementation. Do you think it's doable?

I have made the change as you suggested

> Also, would you please move this protected function to be after the public ones and overriden protected ones in all 4 classes?

I moved this method after the public ones. But I also see that there are a lot of public, protected, private methods in `IcebergDistributedSmokeTestBase.java` that are jumbled up.